README.md

```diff
@@ -0,0 +1,24 @@
+semver-aas
```

Comment:
Yeah, I find `-aas` rather cryptic and too close to another word matching `/a.s/` regexp ;) How about "semver-server" or "semver-service"? Just don't abbreviate to [SS](https://en.wikipedia.org/wiki/Schutzstaffel).